lib/ssh/src/ssh_client_channel.erl

```diff
@@ -216,8 +214,6 @@ call(ChannelPid, Msg) ->
     call(ChannelPid, Msg, infinity).
 
 -doc """
-call(ChannelRef, Msg, Timeout) -> Reply | {error, Reason}
```

Comment:
while at it. reply/2 and enter_loop/1 docs look strange. can you adjust them as well?

![image](https://github.com/user-attachments/assets/a1e048eb-a2ed-41e1-a859-5e8da33bfdcd)

below funny italic also is rendered as we have text between `_` characters.
![image](https://github.com/user-attachments/assets/8bab3c70-31f9-4ef5-82ee-e4c6cff4412a)